Dockerfile.dev

```diff
@@ -0,0 +1,22 @@
+# This docker file build a editable image.
+# You could directly edit the source code at `/home/brkraw to make changes. Or you could mount your brkraw code folder to /home/brkraw
+
+FROM python:3.7
+RUN apt-get update && apt-get install -y git
+RUN mkdir /data
+WORKDIR /home
+RUN git clone https://github.com/BrkRaw/brkraw.git
+WORKDIR /home/brkraw
+RUN git remote rename origin brkraw
+WORKDIR /
+RUN python -m pip install -e /home/brkraw
```

Comment:
I don't agree with all this cloning. Since this Dockerfile lives next to all the files in the github, the idea should be to copy the local files into the container and install from there.

That will also eliminate the need for the git install.

Reply:
This is indeed a good point. I have changed to the way you want, then realized that that's not a good way. This is a dev version where git is needed. You can make changes and use git to push. Also, if you copy your local files rather than from the remote, you also copied the local changes. If you copy the fresh from the remote, everything is in sync. 
In addition, if you want your local changes reflected in the docker image, you simply override it with `-v <your brkraw folder>:/home/brkraw` I think it works better this way. 

I also have changed the workdir here.

Reply:
The whole idea of a Dockerfile.dev is to be able to build a version from the current working directory of git, which may have uncommitted changes.

Reply:
> The whole idea of a Dockerfile.dev is to be able to build a version from the current working directory of git, which may have uncommitted changes.

I see your idea. We had different picture of the dev purpose. Anyhow, You have longer term of invested interest in it and I let you make the call.  I will make the change.